src/Makefile.test.include

```diff
@@ -84,7 +84,9 @@ BITCOIN_TEST_SUITE = \
 if ENABLE_WALLET
 BITCOIN_TEST_SUITE += \
   wallet/test/wallet_test_fixture.cpp \
-  wallet/test/wallet_test_fixture.h
+  wallet/test/wallet_test_fixture.h \
+  wallet/test/pos_test_fixture.cpp \
+  wallet/test/pos_test_fixture.h
```

Comment:
this fixture is going to be used only in `pos_validations_tests.cpp`.
Imo it's not worth to define it in it's own header/cpp file. Can directly define it inside `pos_validation_tests.cpp`, as it's not likely to be used in other test files.

Reply:
That is a slightly different fixture though: activation heights of POS/Shield enforcement, and snark parameters initialization.

Defining this new fixture directly inside the test file actually reduces code churn.
And IMO it's fine to keep these two fixtures separated (even though they're very similar). 
If there's any reason to make `wallet_sapling_transactions_validations_tests` and `pos_validations_tests`  share the same fixture (defined in it's own file), it can be done later, in a dedicated test/refactoring PR.

No big deal anyway.